operatorcourier/validate.py

```diff
@@ -467,7 +468,7 @@ def get_alm_kinds(alm_examples):
         # capabilities check
         if not is_capability_level(annotations["capabilities"]):
             self._log_error("metadata.annotations.capabilities %s is not a "
-                            "valid capabilities level", annotations["capability"])
+                            "valid capabilities level", annotations["capabilities"])
             valid = False
 
         return valid
```

Comment:
The three changes in this file all seem unrelated. Can you please submit them as separate pull requests?

Reply:
Sure thing!